LICENSE.md

```diff
@@ -424,6 +424,31 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+
+### ESLint
```

Comment:
I thought "better safe than sorry" and was initially intending to replace JSHint as it is used with Sandcastle.

I will remove ESLint from the license (at least for now)